README.md

```diff
@@ -1 +1,42 @@
-# Hello_Microverse_Project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> One paragraph statement about the project.
+This is my first Microverse Project
+
+## Built With
+
+- HTML
+- CSS
+- Technologies used
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+
+## Authors
+
+👤 **Author1**
```

Comment:
- [ ] Feel free also in this section to add your github handle in between the square brackets like this [@OLIPLICHE] instead of [@githubhandle] and [olipliche-paka-mavoungou] besides your LinkedIn url instead of just [LinkedIn]